packages/collector/test/tracing/sdk/rootExitSpans/test.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+const expect = require('chai').expect;
+const path = require('path');
+const semver = require('semver');
+
+const supportedVersion = require('@instana/core').tracing.supportedVersion;
+const config = require('@instana/core/test/config');
+const ProcessControls = require('../../../test_util/ProcessControls');
+const globalAgent = require('../../../globalAgent');
+const { retry, delay, expectExactlyOneMatching } = require('@instana/core/test/test_util');
+const constants = require('@instana/core').tracing.constants;
+
+const mochaSuiteFn =
```

Comment:
QS: We are still supporting Node.js 14 and 16. Why are we choosing native fetch here? We've already tested native fetch with this feature in other tests. Can we consider using `got` instead of fetch?

Reply:
let me check that :)

Reply:
> Warning: This package is native [ESM](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) and no longer provides a CommonJS export. If your project uses CommonJS, you will have to [convert to ESM](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c). Please don't open issues for questions regarding CommonJS / ESM.

> 


> You probably want [Ky](https://github.com/sindresorhus/ky) instead, by the same people. It's smaller, works in the browser too, and is more stable since it's built upon [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).

Reply:
As long as we support 14/16, we should not use native fetch in our test env.
Please use the same library we use for all other tests.

Furthermore: Please rename the folder from `rootExitSpans` to `allowRootExitSpans`. Thanks

Reply:
@abhilash-sivan Resolved?

Reply:
Resolved with `node-fetch-v2` as in other tests.
Renaming is also completed.